hazelcast/src/main/java/com/hazelcast/config/CardinalityEstimatorConfig.java

```diff
@@ -61,18 +70,25 @@ public CardinalityEstimatorConfig(String name) {
     }
 
     public CardinalityEstimatorConfig(String name, int backupCount, int asyncBackupCount) {
-        this(name, backupCount, asyncBackupCount, "");
+        this(name, backupCount, asyncBackupCount, DEFAULT_MERGE_POLICY_CONFIG);
     }
 
-    public CardinalityEstimatorConfig(String name, int backupCount, int asyncBackupCount, String quorumName) {
+    public CardinalityEstimatorConfig(String name, int backupCount, int asyncBackupCount, MergePolicyConfig mergePolicyConfig) {
```

Comment:
Do we need just this variant with the additional `MergePolicyConfig` parameter? Should we remove it? Should we add others like `public CardinalityEstimatorConfig(String name, int backupCount, int asyncBackupCount, String quorumName)`?

Reply:
The new variant is needed for the copy constructor. For now I guess we don't need one for quorum only, we can always extend as we see fit.